maven_install_2.13.json

```diff
@@ -1,8 +1,8 @@
 {
     "dependency_tree": {
         "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": "THERE_IS_NO_DATA_ONLY_ZUUL",
-        "__INPUT_ARTIFACTS_HASH": 9508128,
-        "__RESOLVED_ARTIFACTS_HASH": 1793136855,
+        "__INPUT_ARTIFACTS_HASH": 1682121730,
```

Comment:
Making sure you intentionally updated this file, rather than it being changed by accidental build system shenanigans:

Reply:
This one was intentional, added circe optics.